### ISSUE TITLE
CoC mention prestashop.com contact form for how to deal with abuse

### ISSUE THREAD

**Issue by matks** [PrestaShop/open-source#171]
In http://www.prestashop-project.org/code-of-conduct/ section "Enforcement"

> Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at https://www.prestashop.com/en/abuse. 

It would be cool to provide another way to reach out 👍 outside of prestashop.com

**Comment by coffrep** [comment]
Hi @matks you are right about the PrestaShop SA link, we will look into it and see what we can do. Including @kpodemski in this comment.

**Comment by kpodemski** [comment]
@coffrep @eternoendless 

maybe an email under prestashop-project.org? unless it's absolutely required to have a form available

**Comment by coffrep** [comment]
Creating an alias email would definitely work - @eternoendless do let us know of your thoughts. I can help with the creation of the email alias if it is needed.

**Comment by eternoendless** [comment]
I don't think we can get emails @prestashop-project.org, but we could ask for an email alias like `oss-abuse` at prestashop.com

**Comment by coffrep** [comment]
Perfect - @eternoendless let's confirm the email alias name tomorrow and then ask for its creation

**Comment by coffrep** [comment]
The OSS abuse email alias has now been created by the IT team: oss-abuse@prestashop.com

**Comment by matks** [comment]
Thank you!
